scripts/publish.sh

```diff
@@ -52,22 +52,22 @@ Remember to create a release on GitHub with a changelog notes:
 EOF
 }
 
-# Show help if no arguments passed
-if [ $# -eq 0 ]; then
-  echo "Error! Missing release type argument"
-  echo ""
-  echo_help
-  exit 1
+if [ $# -eq 1 ]; then
```

Comment:
You probably want `-gt 0` here otherwise multiple arguments will bypass this check